### PR TITLE
Smoke test protobuf plugin

### PR DESCRIPTION
**Pull request by eskatos**

### CODE REVIEW

File: subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy [hunk 1]
```diff
@@ -436,4 +437,74 @@ class ThirdPartyPluginsSmokeTest extends AbstractSmokeTest {
...
+                id('java')
+                id("com.google.protobuf") version "${TestedVersions.protobuf}"
+            }
+
+            ${mavenCentralRepository()}
+
+            def protobufVersion = '3.11.1'
+            protobuf {
+                protoc {
+                    artifact = "com.google.protobuf:protoc:${'$'}protobufVersion"
```
Comment (lptr) [inline subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy]: Use `\$` instead?

File: subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy [hunk 2]
```diff
@@ -436,4 +437,74 @@ class ThirdPartyPluginsSmokeTest extends AbstractSmokeTest {
...
+        given:
+        buildFile << """
+            plugins {
+                id('java')
+                id("com.google.protobuf") version "${TestedVersions.protobuf}"
+            }
+
+            ${mavenCentralRepository()}
+
+            def protobufVersion = '3.11.1'
```
Comment (lptr) [inline subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy]: Should this be a parameter in `TestedVersions`?

File: subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy (hunk 1 above)
Comment (eskatos) [reply subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy]: Heh, I'm too used to Kotlin I guess :)
Anyway, after other changes this is now irrelevant.

File: subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy (hunk 2 above)
Comment (eskatos) [reply subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ThirdPartyPluginsSmokeTest.groovy]: Good point, done.

**Pull request merged**